src/chronify/models.py

```diff
@@ -44,9 +44,25 @@ def check_columns(cls, columns: list[str]) -> list[str]:
             _check_name(column)
         return columns
 
+    @model_validator(mode="before")
+    @classmethod
+    def update_time_array_id_columns(cls, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Need help here as it is a total hack. Also the update doesn't seem to stick and get reflected in list_columns(). I've tried model_validator(mode="after") but it would get into a "death" loop.